install-dev/data/xml/quick_access.xml

```diff
@@ -12,7 +12,7 @@
       <link>index.php?controller=AdminCartRules&amp;addcart_rule</link>
     </quick_access>
     <quick_access id="New_product" new_window="0">
-      <link>index.php/product/new</link>
+      <link>/admin-dev/index.php/product/new</link>
```

Comment:
I agree the folder name is not fixed, even the `/new` is not fixed also.
Ok I will think about what @aleeks said.
Thanks ..